src/components/MainComponent.vue

```diff
@@ -1,19 +1,17 @@
 <template>
   <div id="maincontainer">
+    <rules-modal id="rulesModal" class="modal fade rules" tabindex="-1" role="dialog" aria-labelledby="" aria-hidden="true"></rules-modal>
     <div id="header">
       <p>Programming Wars</p>
       <div style="margin-left: auto; padding: 0 10px 0 0">
       <label class="checkbox-inline"><input type="checkbox" value="true" v-model="tipsToggle" checked @click="toggleTipBox">TUTORIAL</label>
         </div>
         <div id="header-buttons">
         <button class="btn btn-primary"><router-link to="/">New Game</router-link></button>
+        <button class="btn btn-primary" data-toggle="modal" data-target=".rules">Rules</button>
+        <button class="btn btn-primary" @click="showCredits">Credits</button>
 
-        <button class="btn btn-primary" v-on:click="showCredits">
-        Credits
-      </button>
-        <a class="btn btn-primary" href="https://github.com/sscullen/program-wars/issues/new" target="_blank">
-        Report Issue
-      </a>
+        <a class="btn btn-primary" href="https://github.com/sscullen/program-wars/issues/new" target="_blank">Report Issue</a>
```

Comment:
Looks like this needs to changed to https://github.com/johnanvik/program-wars/issues